src/test/java/picard/util/GCloudTestUtils.java

```diff
@@ -0,0 +1,61 @@
+package picard.util;
+
+public final class GCloudTestUtils {
+    /**
+     *  This is a public requester pays bucket owned by the broad-gatk-test project.
+     *  It must be owned by a different project than the service account doing the testing or the test may fail because it can access the
+     *  file directly through alternative permissions.
+     */
+    public static final String REQUESTER_PAYS_BUCKET_DEFAULT = "gs://hellbender-requester-pays-test/";
+
+    public static final String TEST_INPUTS_DEFAULT = "gs://hellbender/test/resources/";
+    public static final String TEST_STAGING_DEFAULT = "gs://hellbender-test-logs/staging/";
+    public static final String TEST_PROJECT_DEFAULT = "broad-dsde-dev";
+
+
+    /**
+     * A publicly readable GCS bucket set as requester pays, this should not be owned by the same project that is set
+     * as {@link #getTestProject()} or the tests for requester pays access may be invalid.
+     *
+     * @return PICARD_REQUESTER_PAYS_BUCKET env. var if defined, {@value GCloudTestUtils#REQUESTER_PAYS_BUCKET_DEFAULT}.
+     */
+    public static String getRequesterPaysBucket() {
+        return getEnvironmentVariable("PICARD_REQUESTER_PAYS_BUCKET", REQUESTER_PAYS_BUCKET_DEFAULT);
+    }
+
+    private static String getEnvironmentVariable(final String variableName, final String defaultValue) {
+        final String valueFromEnvironment = System.getProperty(variableName);
```

Comment:
This is probably copied from somewhere else, but its  always find it a bit confusing to have a method named `getEnvironmentVariable` that actually gets a JVM property.

Reply:
Actually, are those values actually env variables, or properties ?

Reply:
That is confusing.  I think that's how it was named originally unless I mangled it more here.  Java exposes all the environment variables as system properties, so in this case it's both I guess.

Reply:
I didn't realize Java did that, but it explains a lot of confusion I have previously encountered.